ecosystem/sep-0006.md

```diff
@@ -589,29 +793,33 @@ The response should be a JSON object like:
 }
 ```
 
-The JSON object contains an entry for each Stellar asset that the anchor supports for SEP6 deposit and/or withdrawal.
+The JSON object contains an entry for each **Stellar asset** that the anchor supports for SEP-6 deposit and/or withdrawal.
```

Comment:
❔  What's the motivation for bolding some terms throughout the doc? I think where we have seen emphasis work really well is to emphasis entities only, for example the Server and Client. I think when we start emphasizing other things it becomes difficult to read the doc. 

Thoughts @marcelosalloum @JakeUrban ?

Reply:
Yep, that makes sense. I emphasized some parts of the text that I considered key concepts, but this is probably just a bias I have when taking notes to myself. I'll go easier on the habit of emphasizing things.

Reply:
Removed a few uses of `**` in https://github.com/stellar/stellar-protocol/pull/973/commits/70f1a688bf23566df14f418e4c621c0fa2fa95f9.